src/modules/rating/database/ratingscheduler.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable, Logger } from '@nestjs/common'
```

Comment:
**RatingScheduler** is a service. Move the file into a folder named **services** instead.